apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -204,6 +206,28 @@ export const EventSetupTab = (
             )}
           </ul>
         )}
+        {validLocations.some((location) => location.type === "integrations:google:meet") && (
+          <div className="flex">
+            <Icon.FiAlertTriangle className="mr-4" />
+            <p className="text-sm">
+              In order to use Google Meet you must set your{" "}
```

Comment:
@joeauyeung please don't forget to extract strings into t("in_order_to_use_google")